ambry-api/src/main/java/com/github/ambry/config/RouterConfig.java

```diff
@@ -534,6 +536,20 @@
   @Default("com.github.ambry.router.OperationController")
   public final String operationController;
 
+  /**
+   * Implementation class for StoreKeyConverterFactory
```

Comment:
Could you mention that these configs are specific to the embedded AmbryRequests in cloud router only

Reply:
Sure, updated.